releases/nightly-build/compose-files/Makefile

```diff
@@ -41,7 +41,7 @@ portainer-down:
 	docker-compose -p portainer -f docker-compose-portainer.yml down
 
 build:
```

Comment:
Can we move make build up to compose builder and eliminate the target here?

Reply:
Yes, It is currently in both makefiles and now with the move, I agree it is no longer needed at this "user" level.